### PR TITLE
JavaScript function to reverse the odd number in an array

### PR DESCRIPTION
**Pull request by emmakolade**
JavaScript function to reverse the odd number in an array

### CODE REVIEW

File: transform-array.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const arrNum = new Array(2, 1, 4, 3, 6, 5, 8, 7, 10, 9);
+
+// function that double the odd number in an array and reverse the position
+const transformArrayOfNumbers = (arrNum) => {
+  const oddNum = arrNum.filter((num) => num % 2 === 1);
+  const doubleNum = oddNum.map((num) => num * 2);
+  const reverseNum = doubleNum.reverse();
```
Comment (gwa-admin) [inline transform-array.js:7]: The only one comment I have is that you could have returned from here and saved an extra variable, so something like `return doubleNum.reverse()`

**Review by gwa-admin — COMMENTED** [review]
Really great job on this task @emmakolade, very impressive! 👍

Here's a summary:

**What you did well.**
- Good variable names.
- Demonstrated very good knowledge of JavaScript array methods with use of `reverse` and of functional programming methods like `map` and `filter`.
- Robust code that handled edge cases because when I passed `[1, 10, 5, null, undefined, 's', 5]` as input to your function, it worked.

**Things you can do better on**
For this task, honestly not a lot, the comment I added is the only area of improvement. This was really impressive!

File: transform-array.js (hunk 1 above)
Comment (emmakolade) [reply transform-array.js:7]: Thank you Sir.

**Comment by gwa-admin** [comment]
Hello @emmakolade, closing this PR as part of cleanup work in preparation for our project. Just FYI.

**Pull request closed without merging**